### PR TITLE
Don't build all themes in TEST_CI mode

### PR DESCRIPTION
**Pull request by AlekseyMartynov**
We used to do this in the old ~good~ style-compiler: 
https://github.com/DevExpress/DevExtreme/blob/26dc16cfc6b742e0f4dc9c3e7531b7177bb039d2/build/style-compiler/LessAggregation.cs#L207

Let's revive.

### CODE REVIEW

File: build/gulp/style-compiler.js [hunk 1]
```diff
@@ -20,6 +20,15 @@ const autoPrefix = new LessAutoPrefix({ browsers: browsersList });
 const cssArtifactsPath = path.join(process.cwd(), 'artifacts', 'css');
 const commentsRegex = /\s*\/\*[\S\s]*?\*\//g;
 
+const DEFAULT_DEV_BUNDLE_NAMES = [
+    'common',
+    'light',
+    'material.blue.light',
+    'ios7.default'
```
Comment (babich-a) [inline build/gulp/style-compiler.js:27]: We don't need ios theme in any ci tasks.

**Review by babich-a — CHANGES_REQUESTED** [review]
Is there a place where we check that all themes are compiled? Some generic bundles are different from others. Or the nightly build will be enough?

File: build/gulp/style-compiler.js (hunk 1 above)
Comment (AlekseyMartynov) [reply build/gulp/style-compiler.js:27]:
```
Tests run: 4325, Failures: 12, Not run: 0
--------------------------------------------------------------------------------
DevExpress.knockout/fieldset_bundled.ios7.tests.js - Horizontal align for different widgets on field-value
DevExpress.knockout/fieldset_bundled.ios7.tests.js - Equal width for same widgets on and in field-value
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxTextbox on Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxTextbox in Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxAutocomplete on Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxAutocomplete in Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxLookup on Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxLookup in Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxNumberbox on Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxNumberbox in Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxTextarea on Field
DevExpress.knockout/fieldset_bundled.ios7.tests.js - dxTextarea in Field

```

**Comment by AlekseyMartynov** [comment]
> Is there a place where we check that all themes are compiled? Some generic bundles are different from others. Or the nightly build will be enough?

I think our nightly build does the job. If necessary, we can add a single target to check the complete Gulp build.

**Review by babich-a — APPROVED** [review]

**Review by AntonSermyazhko — APPROVED** [review]

**Review by San4es — APPROVED** [review]

**Pull request merged**